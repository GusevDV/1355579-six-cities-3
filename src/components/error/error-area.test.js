import React from "react";
import renderer from "react-test-renderer";
import ErrorArea from "./error-area.jsx";

it(`Should Throbber component render correctly`, () => {

  const render = renderer.create(
      <ErrorArea message={`Error`}/>
  )
  .toJSON();

  expect(render).toMatchSnapshot();
});
