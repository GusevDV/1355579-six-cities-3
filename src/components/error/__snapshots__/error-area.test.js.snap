// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Throbber component render correctly 1`] = `
<p
  className="error"
>
  Error
</p>
`;
